### DISCUSSION TITLE
Shared type global scope - is this shared across projects?

### DISCUSSION THREAD

**Discussion by richardoSGSI** [thomhurst/TUnit#931] [Q&A] [answered]
When reading [this page](https://thomhurst.github.io/TUnit/docs/tutorial-basics/class-data-source) in the documentation, the `Shared = SharedType.Globally` states that the instance is shared "globally" for every test that also uses this setting.

Does globally indicate that the instance is shared across the project or across a solution?

**Comment by thomhurst** [comment, marked as answer]
It means every test that will be executed in your test session. So basically the lifetime of the `dotnet` process of your tests.

So if multiple projects end up running tests, it'll be shared across those. There is a shared per assembly option if you want to split things out.
